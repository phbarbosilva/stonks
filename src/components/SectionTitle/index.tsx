import React from 'react'
import styled from 'styled-components'

function SectionTitle({ className, children }) {
  return <p className={className}>{children}</p>
}

export default styled(SectionTitle)`
  font-size: 18px;
  font-weight: 700;
  color: var(--blue-1);
`
