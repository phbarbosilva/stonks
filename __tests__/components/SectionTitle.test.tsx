import React from 'react'

import { render } from '@testing-library/react'

import SectionTitle from '../../src/components/SectionTitle'

describe('<SectionTitle />', () => {
  test('snapshot testing', async () => {
    let { asFragment } = render(<SectionTitle>Title</SectionTitle>)

    expect(asFragment()).toMatchSnapshot()
  })
})
