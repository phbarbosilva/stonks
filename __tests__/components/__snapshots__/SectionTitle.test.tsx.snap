// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionTitle /> snapshot testing 1`] = `
<DocumentFragment>
  <p
    class="SectionTitle-sc-1jv5d6w-0 fkPjFP"
  >
    Title
  </p>
</DocumentFragment>
`;
